lib/Service/TokenService.php

```diff
@@ -311,4 +315,44 @@ public function getExchangedToken(string $targetAudience): Token {
 			throw new TokenExchangeFailedException('Failed to exchange token, error in the exchange request', 0, $e);
 		}
 	}
+
+	/**
+	 * Try to get a token from the Oidc provider app for a user and a specific audience (client ID)
+	 *
+	 * @param string $userId
+	 * @param string $targetAudience
+	 * @return Token|null
+	 */
+	public function getTokenFromOidcProviderApp(string $userId, string $targetAudience): ?Token {
+		if (!class_exists('OCA\OIDCIdentityProvider\AppInfo\Application')) {
```

Comment:
```suggestion
		if (!class_exists(\OCA\OIDCIdentityProvider\AppInfo\Application::class)) {
```

This is fine in PHP even if the app is not loaded / available.